libeos-payg/tests/manager.c

```diff
@@ -617,6 +705,30 @@ main (int    argc,
      GUINT_TO_POINTER (TEST_MANAGER_DISABLED_REMOVE_KEY |
                        TEST_MANAGER_DISABLED_SET_ENABLED_PROPERTY));
   T ("/manager/load-empty", test_manager_load_empty, NULL);
+
+  const char *full_matches[] = { "00000000", "12345678", NULL };
+  add_many ("/manager/code-format/matches",
+            test_manager_code_format_matches,
+            full_matches);
+
+  const char *partial_matches[] = { "1", "12", "1234567", NULL };
+  add_many ("/manager/code-format/partial",
+            test_manager_code_format_matches_partial,
+            partial_matches);
+
+  const char *non_matches[] = {
+      "\n", "a", "1a", "123\n", "a123",
+      /* '6' in various non-ASCII forms */
+      "६", "೬", "𝟨", "六",
```

Comment:
I can see you’ve had a lot of fun here. This review has been a rollercoaster of emotion.